dev/com.ibm.ws.jsf.2.2/bnd.bnd

```diff
@@ -78,6 +82,7 @@ Export-Package: \
   com.ibm.ws.jsf.config.*, \
   com.ibm.ws.jsf.ext, \
   com.ibm.ws.jsf.extprocessor, \
+  com.ibm.ws.jsf.fips.*, \
```

Comment:
As far as I can see there is only one package , can we just export `com.ibm.ws.jsf.fips`?

Reply:
Oops. Fixed!